crowdsourcing/models.py

```diff
@@ -1,13 +1,15 @@
 from django.contrib.auth.models import User
 from django.db import models
+from model_utils import Choices
```

Comment:
it is the right way to handle Choices although Im not convinced that it's worth adding a dependency
same thing can be achieved without it 
```
class ModelEg(Model):
   PROJECT_CREATED = 1 
   PROJECT_PUBLISHED = 2
   statuses = ((PROJECT_CREATED, 'Created'),
                (PROJECT_PUBLISHED, 'Published'), )

```
or is there something more to it?